packages/manager/modules/pci/src/components/project/storages/index.js

```diff
@@ -1,9 +1,7 @@
 import angular from 'angular';
 
-import textAccordion from './databases/text-accordion';
-
 const moduleName = 'ovhManagerPciComponentsStorages';
 
-angular.module(moduleName, [textAccordion]);
+angular.module(moduleName);
```

Comment:
Does it work like that ? If I'm not wrong, without giving an array of dependencies (as least an empty one) you're calling an existing module and declaring a new one

Reply:
it looks work correctly on my local but I will update it to be sure